src/latex2sympy/units/unit_aliases/unit_aliases.py

```diff
@@ -315,39 +278,47 @@ def get_aliases_for_unit(unit, dir_module=None, attr_name=None):
             UNIT_ALIASES[alias] = u
 
 # -------------------------------------------------------------------------------------------------
-# test output
+# output pickled file
 # -------------------------------------------------------------------------------------------------
 
-# ALIASES_BY_UNIT = {}
-# for alias, unit in UNIT_ALIASES.items():
-#     base_sympy_unit = get_base_unit(unit, sympy_units)
-#     base_unit = get_base_unit(unit, additional_units) if base_sympy_unit is None else base_sympy_unit
-#     unit_name = str(unit.name) if base_unit is None else str(base_unit.name)
+ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
+
+pickle_file = open(f'{ROOT_DIR}/unit_aliases.pkl', 'wb')
+pickle.dump(UNIT_ALIASES, pickle_file)
+pickle_file.close()
+
+# -------------------------------------------------------------------------------------------------
+# output JSON debug file
+# -------------------------------------------------------------------------------------------------
 
-#     unit_obj = ALIASES_BY_UNIT.get(unit_name, {})
-#     aliases = unit_obj.get('aliases', [])
+ALIASES_BY_UNIT = {}
```

Comment:
@builtbywill Just checking, is this output debug file section supposed to be commented out, or active in the commit? Or does it even matter?

Reply:
@tragessere I uncommented it since this file isn't imported and ran all the time, so if do run/compile it, it means you should probably be checking the json output